_15thnight/core.py

```diff
@@ -95,10 +96,26 @@ def respond_to_alert(provider, needs_provided, alert):
     return response
 
 
-def send_out_resolution(need):
-    """
-    Sends out a resolution for a need to providers.
-    """
+def send_alert_resolution_notice(need):
+    # Check if alert is closed and trigger an alert closed notification
+    if not need.alert.is_open():
+        alert = need.alert
+        body = "Alert sent at {0} for a {1}yo " % (alert.created_at, alert.age)
+        body += "{0} " % alert.gender if alert.gender.lower() in [
+            "female", "male"] else ""
+        body += (
+            "with {0} needs has been closed.\nSee: {1}" %
+            (alert.need_count(), alert.url)
+        )
+        providers = [provider for provider in alert.providers_notified]
+        subject = "Alert %s has closed." % alert.id
+
+        for provider in providers:
+            queue_send_message.apply_async(args=[
+                provider.email, provider.phone_number, subject, body])
+
+
+def resolve_need(need):
     alert = need.alert
```

Comment:
Put a multiline comment here that says: `Resolve a need and set alert as closed when all needs are resolved`